prompt-management/getting-started.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: "Get Started"
+description: "Create your first prompt and use it in your application"
+---
+
+Learn how to create your first prompt in LangWatch and use it in your application with dynamic variables. This enables your team to update AI interactions without code changes.
+
+## Get API keys
+
+1. Create a LangWatch [account](https://app.langwatch.ai) or set up [self-hosted LangWatch](https://github.com/langwatch/langwatch?tab=readme-ov-file#self-hosted-%EF%B8%8F)
+2. Create new API credentials in your [project settings](https://app.langwatch.ai/settings)
+3. Note your API key for use in the steps below
+
+## Create a prompt
+
+<Tabs>
+  <Tab title="LangWatch UI">
+    Use the LangWatch UI to create a new prompt or update an existing one.
+
+    1. Navigate to your project dashboard
+    2. Go to **Prompt Management** in the sidebar
+    3. Click **"Create New Prompt"**
+    4. Fill in the prompt details and save
+
+    <Frame>
+      <img
+        className="block"
+        src="/images/prompts/view-editing-the-prompt.png"
+        alt="Editing a prompt in LangWatch UI"
+      />
+    </Frame>
+
+  </Tab>
+
+  <Tab title="REST API">
+    Use the REST API to create a new prompt:
+
+    ```bash create_prompt.sh
+    # Create a new prompt (this creates the prompt with an initial version)
+    curl -X POST "https://app.langwatch.ai/api/prompts" \
+      -H "Content-Type: application/json" \
+      -H "X-Auth-Token: your-api-key" \
+      -d '{
+        "name": "customer-support",
+        "handle": "customer-support-bot",
+        "scope": "project"
```

Comment:
1. skip name
2. shouldn't scope be "PROJECT" all caps?
3. feels weird to have that without having any actual prompt content at all? shouldn't this api accept somewhere for me to put my messages with a system message saying "you are a helpful assistant" or something?

Reply:
@rogeriochaves we don't support creating a version with the prompt yet via the API or any of the sdk implementations.

Reply:
ok, that's a TODO then for the next PRs